charts/prometheus-mongodb-exporter/templates/deployment.yaml

```
@@ -60,6 +60,10 @@ spec:
             {{- toYaml .Values.volumeMounts | nindent 12 }}
       affinity:
         {{- toYaml .Values.affinity | nindent 8 }}
+      {{- if .Values.dnsConfig }}
+      dnsConfig:
+        {{- toYaml .Values.dnsConfig | nindent 8 }}
```

Thank you, @calvinbui, for your PR.

```suggestion
      {{- with .Values.dnsConfig }}
      dnsConfig:
        {{- toYaml . | nindent 8 }}
```